efficientdet/det_model_fn.py

```diff
@@ -413,23 +379,8 @@ def model_fn(inputs):
 
     if is_tpu:
       optimizer = tf.tpu.CrossShardOptimizer(optimizer)
-    if params['device']['grad_ckpting']:
-      # pylint: disable=g-import-not-at-top,g-direct-tensorflow-import
-      from third_party.grad_checkpoint import grad
-      from tensorflow.python.ops import gradients
-      # pylint: enable=g-import-not-at-top,g-direct-tensorflow-import
-
-      # monkey patch tf.gradients to point to our custom version,
-      # with automatic checkpoint selection
-      def gradients_(ys, xs, grad_ys=None, **kwargs):
-        return grad.gradients(
-            ys,
-            xs,
-            grad_ys,
-            checkpoints=params['device']['grad_ckpting_list'],
-            **kwargs)
-
-      gradients.__dict__['gradients'] = gradients_
+    elif params['mixed_precision']:
+      optimizer = tf.train.experimental.enable_mixed_precision_graph_rewrite(optimizer)
```

Comment:
Should gain better performance and less bugs.
Fix https://github.com/google/automl/issues/852